### DISCUSSION TITLE
Sharing the Security Hub with non-administrators

### DISCUSSION THREAD

**Discussion by tostt** [goharbor/harbor#19233] [Ideas]
The Security Hub introduced in v2.9.0-rc1 is an exciting feature that I am looking forward to using in production.
This hub is currently only available to Admin users.

It would be interesting to share this hub with people who are not Admins, such as a CISO (Chief Information Security Officer) who doesn't need (and prefers not to have, for safety reasons) Admin rights.
This could be a new role, perhaps a special type of "Guest".

**Comment by dioguerra** [comment]
I would also like this feature, and leveraging the meta of deploying with 2.10 enhanced robot accounts this could also be taken into account.

This user `CISO` or user group would have access to the security dashboard as currently the `admin` user has.

**Comment by antoinelrya** [comment]
It would be interesting to have the option to define a Security Hub administrator group that is separate from the Harbor administrators.
It makes sense to separate the two roles: one managing CVE analysis, and the other handling the maintenance of the tool.

**Comment by eidmantas** [comment]
Very much would like to see this. I see there was also https://github.com/goharbor/harbor/issues/19594
